dev-tools.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
this is the file that lives inside of projects. we want to make sure that any developer can understand what itis all about by reading it, hence the change